marklogic-data-hub/src/main/java/com/marklogic/hub/dhs/installer/deploy/DeployHubAmpsCommand.java

```diff
@@ -38,12 +39,16 @@ protected String getResourceName() {
 
         @Override
         public SaveReceipt updateResource(String payload, String resourceId) {
+            Versions.MarkLogicVersion mlVersion = new Versions().getMLVersion(getManageClient().getXml("/manage")
```

Comment:
For the path, use "/node()/c:version" since we know exactly where to find the version. This prevents us from ever getting two values back, which would cause a break.

Also, I recommend moving all of this into a "private boolean canUpdateAmps" method. And only call it inside the catch block here - we don't need to check this unless we get an exception. And in that private method, put everything in a try/catch block, as we don't ever want an exception being thrown by the version-check code. Instead, do something like this:

    logger.warn("Could not determine MarkLogic version; cause: " + e.getMessage() + "; will assume that user can update amps and will throw original exception");